### ISSUE TITLE
User API

### ISSUE THREAD

**Issue by samdark** [phpdigest/api#1]
POST /user - создать юзера

```json
{
  token: x
}
```

GET /user?token=x - получить инфу о юзере по токену

```json
{
   id: 42
}
```

**Comment by romkatsu** [comment]
Для создания пользователя точно нужно передавать токен? Обычно он генерируется на стороне сервера. 

Методы создания пользователя и получения должны быть доступны без авторизации?

**Comment by roxblnfk** [comment]
сейчас планируется завязка на браузерный плагин в одну кнопку. Юзеру не нужно регаться и т.д.
По умолчанию он нажимает кнопку (где бы она не была) и отправляет анонимно ссылку.
Токен нужен для понимания, сколько ссылок отправил один и тот же аноним. В будущем это скорее всего будет связано с нормальными профилями

**Comment by samdark** [comment]
> Для создания пользователя точно нужно передавать токен? Обычно он генерируется на стороне сервера.

Да, планировалось изначально в плагине генерить токен, но можно по-другому:

1. Плагин делает реквест на API регает юзера (если в плагине его нет).
2. Получает токен и сохраняется его у себя.

Так даже лучше.

**Comment by samdark** [comment]
> Методы создания пользователя и получения должны быть доступны без авторизации?

Создания - да. Получения - по токену.
